tests/fv3_conf/ccpp_gsd_run_drag_suite.IN

```diff
@@ -0,0 +1,100 @@
+rm -fr INPUT RESTART
+mkdir INPUT RESTART
+if [ $WARM_START = .F. ]; then
+  cp -r @[RTPWD]/FV3_input_data_gsd/FV3_input_data_C96_with_aerosols/* INPUT/
+  ln -s /scratch2/BMC/wrfruc/mtoy/FV3GFS/GWD_static_files/C96/tile1/oro.tile1.nc ./INPUT/oro_data_ls.tile1.nc
```

Comment:
I will need to copy these files into the regression test baseline and ask you to change the paths here. Also, we could add a check in `ccpp_gsd_run.IN` if the GSD drag suite is used (similar to how the switches are made for the microphysics/lsm) instead of duplicating the entire file?

Reply:
Thanks for adding this logic to `tests/fv3_conf/ccpp_gsd_run.IN`. I will remove tests/fv3_conf/ccpp_gsd_run_drag_suite.IN when I pull your code into the latest version of gsd/develop (happening now).